packages/console/src/components/Markdown/index.tsx

```diff
@@ -57,6 +57,15 @@ const Markdown = ({ className, children }: Props) => {
         img: ({ src, alt }) => {
           return <GithubRawImage src={src} alt={alt} />;
         },
+        a: ({ href, children }) => {
+          const isExternalLink = href?.startsWith('http');
+
+          return (
+            <a href={href} target={isExternalLink ? '_blank' : '_self'} rel="noreferrer noopener">
```

Comment:
```suggestion
            <a href={href} target={isExternalLink ? '_blank' : '_self'} rel="noopener">
```

this is cool. update our lint rule if needed

Reply:
https://github.com/silverhand-io/configs/pull/65

Reply:
Would you like me to update the lint rule in configs or set it at project level?

Reply:
Updated